src/Execution/lbsRuntime.py

```diff
@@ -93,21 +105,11 @@ def __init__(self, p, c: dict, order_strategy: str, v=False):
         _, l_min, self.average_load, l_max, l_var, _, _, l_imb = compute_function_statistics(
             self.phase.ranks,
             lambda x: x.get_load())
-        n_w, _, w_ave, w_max, w_var, _, _, w_imb = compute_function_statistics(
+        n_w, _, w_ave, w_max, _, _, _, _ = compute_function_statistics(
```

Comment:
Just 3 out of 8 function outputs in use, maybe it's time to modify the output of `compute_function_statistics`

Reply:
Yes. We should return a structure and create a class for that (`lbsStatisticsContainer` or something like that). Another day. Please create another issue for that. Thanks.